packages/backend/src/graphql/schema.graphql

```diff
@@ -961,3 +949,4 @@ interface MutationResponse {
 
 scalar UInt8
 scalar UInt64
+scalar JSONObject
```

Comment:
Also considered the `JSON` scalar type, but that could be of type string, number, boolean, object, array or null (https://stackoverflow.com/questions/70607882/what-is-the-difference-between-graphqljson-and-graphqljsonobject-exported-by-gra/70667638#70667638) which I didn't think we wanted. `JSONObject` ensures its an object.